plugins/modules/wafv2_web_acl.py

```diff
@@ -86,6 +87,13 @@
           description:
             - Rule configuration.
           type: dict
+    custom_response_bodies:
+      description:
+        - A map of custom response keys and content bodies. Define response bodies here and reference them in the rules by providing
+        - the key of the body dictionary element.
+        - Each element must have a unique dict key and in the dict two keys for I(content_type) and I(content). Needs botocore >= 1.21.0 to work.
+      type: dict
+      version_added: 2.1.0
```

Comment:
@stefanhorning  2.1.0 is already released.  
Next release will be 3.0.0  
But we can also backport it and it will be than 2.2.0

Reply:
No problem, I can just bump the version here.